### PR TITLE
scylla: add 3.26.7 patch

### LINKED ISSUE #81: Patch required for 3.26.7
Missing a patch for this version, currently fails to apply:

```
Test details
Scylla Version: 5.5.0~dev-0.20240317.88119006022d
Build Id: #1516
Driver repository: https://github.com/scylladb/python-driver.git
Test result
Driver version: 3.26.7-scylla protocol: 3
Traceback (most recent call last):
  File "/jenkins/workspace/scylla-master/driver-tests/python-driver-matrix-test/python-driver-matrix/main.py", line 23, in main
    result = Run(python_driver_git=arguments.python_driver_git,
  File "/jenkins/workspace/scylla-master/driver-tests/python-driver-matrix-test/python-driver-matrix/run.py", line 173, in run
    if self._checkout_branch() and self._apply_patch_files() and self._install_python_requirements():
  File "/jenkins/workspace/scylla-master/driver-tests/python-driver-matrix-test/python-driver-matrix/run.py", line 108, in _apply_patch_files
    self._run_command_in_shell(f"git apply --check {file_path}")
  File "/jenkins/workspace/scylla-master/driver-tests/python-driver-matrix-test/python-driver-matrix/run.py", line 98, in _run_command_in_shell
    assert status_code == 0, stderr
AssertionError: (None, b'/bin/bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)\nerror: patch failed: tests/integration/__init__.py:43\nerror: tests/integration/__init__.py: patch does not apply\n')
```

### LINKED ISSUE #82: Patch required for 3.26.8
Could be solved by #81 

Fails to apply:

```
Test details
Scylla Version: 2023.1.7-0.20240402.1f84904a7100
Build Id: #22
Driver repository: https://github.com/scylladb/python-driver.git
Test result
Driver version: 3.26.8-scylla protocol: 3
Traceback (most recent call last):
  File "/jenkins/workspace/enterprise-2023.1/driver-tests/python-driver-matrix-test/python-driver-matrix/main.py", line 23, in main
    result = Run(python_driver_git=arguments.python_driver_git,
  File "/jenkins/workspace/enterprise-2023.1/driver-tests/python-driver-matrix-test/python-driver-matrix/run.py", line 173, in run
    if self._checkout_branch() and self._apply_patch_files() and self._install_python_requirements():
  File "/jenkins/workspace/enterprise-2023.1/driver-tests/python-driver-matrix-test/python-driver-matrix/run.py", line 108, in _apply_patch_files
    self._run_command_in_shell(f"git apply --check {file_path}")
  File "/jenkins/workspace/enterprise-2023.1/driver-tests/python-driver-matrix-test/python-driver-matrix/run.py", line 98, in _run_command_in_shell
    assert status_code == 0, stderr
AssertionError: (None, b'/bin/bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)\nerror: patch failed: tests/integration/__init__.py:43\nerror: tests/integration/__init__.py: patch does not apply\n')
```

### PR DESCRIPTION
**Pull request by k0machi**
Patch also applies for .8, so closing both issues with this.

Fixes #81 
Fixes #82 

[Jenkins](https://jenkins.scylladb.com/job/scylla-staging/job/alexey/job/argus-python-driver-matrix-test/34/)

### CODE REVIEW

**Review by dimakr — APPROVED** [review]
LGTM

**Comment by roydahan** [comment]
Please add a commit message on why this patch was need.

File: versions/scylla/3.26.7/patch.patch [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+         """
+         Test the driver can connect with the no_compact option and the results
+diff --git a/tests/integration/standard/test_custom_cluster.py b/tests/integration/standard/test_custom_cluster.py
+index 6cdfb8d1..46ec7fa1 100644
+--- a/tests/integration/standard/test_custom_cluster.py
++++ b/tests/integration/standard/test_custom_cluster.py
+@@ -26,7 +26,7 @@ def setup_module():
+     config_options = {'native_transport_port': 9046}
+     ccm_cluster.set_configuration_options(config_options)
+     # can't use wait_for_binary_proto cause ccm tries on port 9042
```
Comment (roydahan) [inline versions/scylla/3.26.7/patch.patch:40]: Should you remove this comment now also?
What has changed that we can use it now?

File: versions/scylla/3.26.7/patch.patch [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+ 
+-    @unittest.expectedFailure
+     @requirecassandra
+     @greaterthanorequalcass40
+     def test_protocol_v5_uses_flag_int(self):
+@@ -197,7 +195,6 @@ class CustomProtocolHandlerTest(unittest.TestCase):
+         self._protocol_divergence_fail_by_flag_uses_int(ProtocolVersion.DSE_V1, uses_int_query_flag=True,
+                                                         int_flag=True)
+ 
+-    @unittest.expectedFailure
```
Comment (roydahan) [inline versions/scylla/3.26.7/patch.patch:70]: Are these passing now?
How? don't they need v5 support?

File: versions/scylla/3.26.7/patch.patch [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+     def test_protocol_divergence_v5_fail_by_flag_uses_int(self):
+diff --git a/tests/integration/standard/test_metadata.py b/tests/integration/standard/test_metadata.py
+index c561491a..8c37afa0 100644
+--- a/tests/integration/standard/test_metadata.py
++++ b/tests/integration/standard/test_metadata.py
+@@ -476,7 +476,6 @@ class SchemaMetadataTests(BasicSegregatedKeyspaceUnitTestCase):
+         tablemeta = self.get_table_metadata()
+         self.check_create_statement(tablemeta, create_statement)
+ 
+-    @pytest.mark.skip(reason='https://github.com/scylladb/scylladb/issues/6058')
```
Comment (roydahan) [inline versions/scylla/3.26.7/patch.patch:82]: What has changed here?
I see that this issue is still open...

**Comment by roydahan** [comment]
Maybe I don't understand the goal of the patch.
Now I see that you removed the markers from the tests and moved these tests to the ignore section in the ignore.yaml, but I don't understand why.

**Comment by k0machi** [comment]
> Maybe I don't understand the goal of the patch. Now I see that you removed the markers from the tests and moved these tests to the ignore section in the ignore.yaml, but I don't understand why.

Oh, this is just the previous patch minus the `import ccmlib` declaration that is now part of python-driver tests, I am not exactly sure why do we ignore tests this way, instead of using the skip markers, maybe @fruch can elaborate.

**Review by roydahan — APPROVED** [review]

**Comment by roydahan** [comment]
@k0machi / @fruch ping

**Review by fruch — APPROVED** [review]
LGTM

**Pull request merged**